client/utils/apiWrapper.js

```diff
@@ -103,6 +106,247 @@ export const deleteForm = Model_ID => {
     });
 };
 
+export const register = (emailInput, passwordInput, questionIdx, answer) => {
+  try {
+    return fetch(`${BASE_URL}/auth/register/`, {
+      method: "POST",
+      headers: { "Content-Type": "application/json" },
+      body: JSON.stringify({
+        email: emailInput,
+        password: passwordInput,
+        questionIdx,
+        securityQuestionAnswer: answer,
+        role: "guest",
+        answer
+      })
+    });
+  } catch (err) {
+    console.log(err.message);
+  }
+};
+
+export const login = (emailInput, passwordInput) => {
+  try {
+    return fetch(`${BASE_URL}/auth/login`, {
+      method: "POST",
+      headers: { "Content-Type": "application/json" },
+      body: JSON.stringify({
+        email: emailInput,
+        password: passwordInput
+      })
+    });
+  } catch (err) {
+    console.log(err);
+  }
+};
+
+export const verify = () => {
+  try {
+    return fetch(`${BASE_URL}/auth/verify/`, {
+      method: "POST",
+      headers: { "Content-Type": "application/json", token: getCookie("token") }
+    });
+  } catch (err) {
+    console.log(err);
+  }
+};
+
+export const getSecurityQuestions = () => {
+  try {
+    return fetch(`${BASE_URL}/auth/getSecurityQuestions`, {
+      method: "GET",
+      headers: {
+        "Content-Type": "application/json",
+        token: getCookie("token")
+      }
+    });
+  } catch (err) {
+    return err;
```

Comment:
You can use `return err` like shown here instead of `console.log()` in the api wrapper